pkg/GAPJulia/JuliaInterface/src/convert.c

```diff
@@ -16,10 +18,16 @@ jl_value_t * julia_gap(Obj obj)
         return gap_box_gapffe(obj);
     }
     if (IS_JULIA_OBJ(obj)) {
-        return GET_JULIA_OBJ(obj);
+        BEGIN_GAP_SYNC();
+        result = GET_JULIA_OBJ(obj);
+        END_GAP_SYNC();
+        return result;
     }
     if (IS_JULIA_FUNC(obj)) {
-        return GET_JULIA_FUNC(obj);
+        BEGIN_GAP_SYNC();
+        result = GET_JULIA_FUNC(obj);
```

Comment:
Same remark as above, except that a few more things are read, but all from immutable/read-only objects of type `T_FUNCTION` and `T_JULIA_OBJ`